pkg/log/buffalo_logger.go

```diff
@@ -12,9 +12,13 @@ import (
 // It's also good to note that internal logs
 // from buffalo should only be allowed in development
 // as our logging-system should be handled from our codebase.
-func Buffalo() buffalo.Logger {
+// If the environment is production, then its a noop logger.
+func Buffalo(env string) buffalo.Logger {
 	l := logrus.New()
 	l.Formatter = &buffaloFormatter{}
+	if env == "production" {
+		l.Out = nopwriter{}
+	}
 
```

Comment:
@devdinu we don't need error level from Buffalo, error level is logic based and it comes from our app-logger not buffalo-logger. However, panic makes sense so I've updated the PR. I'm also passing down the level now instead of using ENV.